### PR TITLE
add some golang fake keywords for a better programming experience

### PR DESCRIPTION
**Pull request by chunzhouyu**
hi @dgutov 
plz merge :)

### CODE REVIEW

**Comment by tarsius** [comment]
I don't use, or even maintain, company-mode or go.

**Comment by chunzhouyu** [comment]
@tarsius sorry , i update the merge(r) now ...

**Comment by dgutov** [comment]
The build is still failing.

File: company-keywords.el [hunk 1]
```diff
@@ -142,10 +142,9 @@
       "tiny" "transfer" "transpose" "trim" "true" "type" "ubound" "unpack"
       "use" "value" "verify" "volatile" "wait" "where" "while" "with" "write"))
     (go-mode
-     ;; from https://golang.org/ref/spec#Keywords
```
Comment (dgutov) [inline company-keywords.el]: Could you bring back the comment? And add an extra explanation about the additional "keywords".

Comment (chunzhouyu) [reply company-keywords.el]: 👌

**Comment by chunzhouyu** [comment]
@dgutov i have brought the comment back and because company-go does not support the completion of them yet

**Comment by dgutov** [comment]
> so i added the fake keywords for a better programing experience

Can you add *that* comment?

**Comment by chunzhouyu** [comment]
hey @dgutov, i have added some comments into company-keywords.el, plz review it .

**Comment by dgutov** [comment]
Thanks

**Pull request merged**